src/actions/ScanActions.js

```diff
@@ -268,7 +267,7 @@ export const legacyAddressModalContinueButtonPressed = () => (dispatch: Dispatch
 export const showLegacyAddressModal = () => async (dispatch: Dispatch, getState: GetState) => {
   const legacyAddressModal = createYesNoModal({
     title: s.strings.legacy_address_modal_title,
-    icon: <Icon style={styles.legacyAddressModalIcon} type={ION_ICONS} name={WARNING} size={50} />,
+    icon: <AntDesignIcon name="warning" size={THEME.rem(2)} color={THEME.COLORS.ACCENT_RED} style={{ marginBottom: THEME.rem(0.25) }} />,
```

Comment:
Where does this margin come from ?

Reply:
Answer: It's new, because this used to be an IonIcon, and now it's AntDesignIcon, and the margin makes it look more visually centered.